doc/modules/core.rst

```diff
@@ -316,6 +257,78 @@ Once a :code:`SortingAnalyzer` object is saved to disk, it can be easily reloade
     of failure, for example if the original file is not available, the :code:`SortingAnalyzer`
     will be automatically instantiated in "recordingless" mode.
 
+The :code:`sorting_analyzer` object implements convenient functions to access the underlying :code:`recording` and
+:code:`sorting` objects' information:
+
+.. code-block:: python
+
+    num_channels = sorting_analyzer.get_num_channels()
+    num_units = sorting_analyzer.get_num_units()
+    sampling_frequency = sorting_analyzer.get_sampling_frequency()
+    # or: sampling_frequency = sorting_analyzer.sampling_frequency
+    total_num_samples = sorting_analyzer.get_total_samples()
+    total_duration = sorting_analyzer.get_total_duration()
+    ### NOTE ###
```

Comment:
I don't think this requires a double comment. Or maybe you intended a special formatting that does not seem to be displayed.

Reply:
This is from an older version. I've just gotten rid of the ### NOTE ###.

Reply:
All right.